addOns/graphql/src/main/java/org/zaproxy/addon/graphql/GraphQlGenerator.java

```diff
@@ -0,0 +1,329 @@
+        List<GraphQLArgument> args = field.getArguments();
+        if (!args.isEmpty()) {
+            query.append('(');
+            for (GraphQLArgument arg : args) {
+                query.append(arg.getName())
+                        .append(": ")
+                        .append(getDefaultValue(arg.getType()))
+                        .append(", ");
+            }
+            query.setLength(query.length() - 2);
+            query.append(") ");
+        }
+    }
+
+    private String getDefaultValue(GraphQLType type) {
+        return getDefaultValue(type, 0);
+    }
+
+    private String getDefaultValue(GraphQLType type, int depth) {
+        if (depth > MAX_DEPTH) return "null";
+        StringBuilder defaultValue = new StringBuilder();
+        if (type instanceof GraphQLNonNull) {
+            GraphQLNonNull nonNullType = (GraphQLNonNull) type;
+            type = nonNullType.getWrappedType();
+        }
+        if (type instanceof GraphQLScalarType) {
+            GraphQLScalarType scalar = (GraphQLScalarType) type;
+            switch (scalar.getName()) {
+                case "Int":
+                case "ID":
+                    defaultValue.append("1");
+                    break;
+                case "Float":
+                    defaultValue.append("3.14");
+                    break;
+                case "String":
+                    defaultValue.append("\"hello\"");
+                    break;
+                case "Boolean":
+                    defaultValue.append("true");
+                    break;
+            }
+        } else if (type instanceof GraphQLEnumType) {
+            GraphQLEnumType enumType = (GraphQLEnumType) type;
+            defaultValue.append(enumType.getValues().get(0).getName());
+        } else if (type instanceof GraphQLInputObjectType) {
+            GraphQLInputObjectType object = (GraphQLInputObjectType) type;
+            defaultValue.append("{ ");
+            List<GraphQLInputObjectField> fields = object.getFields();
+            for (GraphQLInputObjectField field : fields) {
+                defaultValue
+                        .append(field.getName())
+                        .append(": ")
+                        .append(getDefaultValue(field.getType(), depth + 1))
+                        .append(", ");
+            }
+            defaultValue.setLength(defaultValue.length() - 2);
+            defaultValue.append(" }");
+        } else if (type instanceof GraphQLList) {
+            GraphQLList list = (GraphQLList) type;
+            String wrappedValue = getDefaultValue(list.getWrappedType(), depth + 1);
+            defaultValue
+                    .append("[")
+                    .append(wrappedValue)
+                    .append(", ")
+                    .append(wrappedValue)
+                    .append(", ")
+                    .append(wrappedValue)
+                    .append("]");
+        } else defaultValue.append("null");
```

Comment:
For consistency this should use curly braces too.
```suggestion
        } else {
            defaultValue.append("null");
        }
```